src/scsiencrypt.cpp

```diff
@@ -369,17 +373,19 @@ bool SCSIExecute(std::string tapedrive, unsigned char *cmd_p, int cmd_len,
 #error "OS type is not set"
 #endif
 #ifdef DEBUGSCSI
-  std::cout << "SCSI Command: ";
+  std::cerr << "SCSI Command: ";
   for (int i = 0; i < cmd_len; i++) {
-    std::cout << std::hex << cmd_p[i];
+    std::cerr << std::hex << std::setw(2) << std::setfill('0')
+              << (unsigned int) {cmd_p[i]} << ' ';
   }
-  std::cout << "\n";
+  std::cerr << std::endl;
```

Comment:
Why do you use `endl` here? Afaik `std::cerr` is unbuffered and flushed with each and every output operation.

Reply:
Fair point; this mostly personal preference and made to match the other case.